spec/client_spec.js

```diff
@@ -226,45 +229,77 @@ describe('Client', function() {
     });
 
     describe('#get', function() {
-      beforeEach(function() {
-        sandbox.spy(window, 'setTimeout');
+      var requestsCount = 0,
+          promise;
+
+      afterEach(function() {
+        promise.catch(function() {});
+        requestsCount++;
       });
 
       it('returns a promise', function() {
-        expect(subject.get('ticket.subject')).to.be.a.promise;
+        promise = subject.get('ticket.subject');
+
+        expect(promise).to.be.a.promise;
       });
 
-      it('rejects the promise after 5 seconds', function() {
-        subject.get('ticket.subject');
-        expect(setTimeout).to.have.been.calledWith(sinon.match.func, 5000);
+      it('accepts an array with multiple paths', function() {
+        promise = subject.get(['ticket.subject', 'ticket.requester']);
+
+        expect(promise).to.be.a.promise;
+      });
+
+      it('rejects the promise after 5 seconds', function(done) {
+        this.timeout(6000);
+
+        promise = subject.get('ticket.subject');
+
+        promise.catch(function(err) {
```

Comment:
We have Chai as Promised, right? Could possibly do something like: `expect(promise).should.be.rejectedWith(Error)`?